### PR TITLE
Add log statement when synchronizing an object with basket

### PR DESCRIPTION
**Pull request by diox**
Extra logging for https://github.com/mozilla/addons-server/issues/12147

### CODE REVIEW

File: src/olympia/users/models.py [hunk 1]
```diff
@@ -962,6 +962,9 @@ def watch_changes(old_attr=None, new_attr=None, instance=None,
     )
     if any(field in changes for field in basket_relevant_changes):
         from olympia.amo.tasks import sync_object_to_basket
+        log.info(
+            'Triggering a sync of %s %s with basket because of %s change',
+            'userprofile', instance.pk, 'attribute')
```
Comment (eviljeff) [inline src/olympia/users/models.py]: Why the unnecessary string substitution in each `log.info`?

Comment (diox) [reply src/olympia/users/models.py]: Just to make sure the string was consistent without having to write a wrapper for it

Comment (eviljeff) [reply src/olympia/users/models.py]: you could have specified the string as a constant.

**Comment by eviljeff** [comment]
I didn't r+ before you merged 😛

**Comment by diox** [comment]
woops sorry about that :)

**Pull request merged**